Documentation/ceph-common-issues.md

```diff
@@ -953,3 +954,61 @@ If you are using Kubernetes version is v1.15 or older, you will see an error lik
 >unable to recognize "STDIN": no matches for kind "CustomResourceDefinition" in version "apiextensions.k8s.io/v1"
 >```
 You need to create the CRDs found in `cluster/examples/kubernetes/ceph/pre-k8s-1.16`. Note that these pre-1.16 `apiextensions.k8s.io/v1beta1` CRDs are deprecated in k8s v1.16 and will no longer be supported from k8s v1.22.
+
+## Unexpected partitions created
+
+### Symptoms
+Unexpected partitions are created on host disks that are used by Ceph OSDs. This happens more often
+on SSDs than HDDs and usually only on disks that are 1TB or larger. Many tools like `lsblk`,
+`blkid`, and `udevadm` will not show a partition table type for the partition; however, `parted`
+will reliably list the partition as `atari` (see the example command and output below).
+
+```console
+# parted --script /dev/sdb1 print
+Model: Unknown (unknown)
+Disk /dev/sdb1: 1509kB
+Sector size (logical/physical): 512B/512B
+Partition Table: atari
+Disk Flags:
+
+Number  Start  End  Size  Type  File system  Flags
+
+```
+
+The underlying issue causing this is Atari (sometimes identified as AHDI) partition support in the
+Linux kernel. Atari partitions have very relaxed specifications compared to modern partition types,
+and it is relatively easy for random data written to a disk to appear as an Atari partition to the
+Linux kernel. Ceph's Bluestore OSDs have an anecdotally high probability of writing data on to disks
+that can randomly appear to the kernel as an Atari partition.
+
+You can see https://github.com/rook/rook/issues/7940 for more detailed information and discussion.
+
+**Rook versions v1.6.0-v1.6.6 may create unwanted OSDs on these partitions, which will corrupt
+existing OSDs.**
```

Comment:
The corruption comes from the OSD being created on the smaller 48Gi partition, right? We should probably release 1.6.7 next week to get this fix out there instead of waiting for the usual cycle.

Reply:
After we get this merged and before the v1.6.7, it would be helpful if someone hitting this issue could confirm the fix in their env.

Reply:
> The corruption comes from the OSD being created on the smaller 48Gi partition, right? We should probably release 1.6.7 next week to get this fix out there instead of waiting for the usual cycle.

Yes. I agree.